client/src/components/CustomProgress.js

```diff
@@ -41,12 +61,52 @@ class CustomProgress extends Component {
           variant="determinate"
           value={value}
           className={classes.progress}
-          classes={{
-            colorPrimary: classes.linearColorPrimary,
-            barColorPrimary: classes.linearBarColorPrimary,
-          }}
+          classes={
```

Comment:
don't need to change this as this is just temp code, but in the future would be nicer to have a lookup table and a factory function